package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "argo-clinical",
-  "version": "1.37.0",
```

Comment:
master is already at  `1.38.0` : https://github.com/icgc-argo/argo-clinical/blob/master/package.json#L3